builtin/builtin.go

```diff
@@ -309,6 +312,37 @@ func builtin_any(self, seq py.Object) (py.Object, error) {
 	return py.False, nil
 }
 
+const ascii_doc = `ascii(obj, /)
+Return an ASCII-only representation of an object.
+
+As repr(), return a string containing a printable representation of an
+object, but escape the non-ASCII characters in the string returned by
+repr() using \\x, \\u or \\U escapes. This generates a string similar
+to that returned by repr() in Python 2
+`
+
+func builtin_ascii(self, o py.Object) (py.Object, error) {
+	reprObject, err := py.Repr(o)
+	if err != nil {
+		return nil, err
+	}
+
+	// TODO(corona10): Replace it into strings.Builder
+	var sb bytes.Buffer
+	repr := reprObject.(py.String)
+	for _, c := range repr {
+		if c <= unicode.MaxASCII {
```

Comment:
This needs to \xXX encode chars in the range 0..31 inclusive with exceptions for \t \r \n

Reply:
@ncw Oh, that's interesting!

Reply:
@ncw 

Could you provide me a sample python code to reproduce it, please?
Unfortunately, I can not reproduce it..

Reply:
```python
# cpython 3.7
>>> ascii(chr(30))
"'\\x1e'"

# current PR

>>> ascii(chr(30))
"'\\x1e'"
```